test/components/views/rooms/__snapshots__/RoomHeader-test.tsx.snap

```diff
@@ -10,9 +10,9 @@ exports[`Roomeader renders with no props 1`] = `
     >
       <div
         aria-level="1"
-        class="mx_RoomHeader_name"
         dir="auto"
         role="heading"
+        style="font: var(--cpd-font-body-lg-semibold); letter-spacing: var(--cpd-font-letter-spacing-body-lg);"
```

Comment:
This switching to inline styles is surprising, why are we bloating the DOM here?

Reply:
I agree with you, it's a bit odd. It's on my list of things to fix on Compound's side. There's more elegant ways to achieve the same thing. 
I however don't think we should block that pull request on this specific thing.

Reply:
> I however don't think we should block that pull request on this specific thing.

I agree in this instance, but I argue it is a blocker to switching to compound components for components used frequently in the DOM, given you get at most 1 room header this is fine

Reply:
https://github.com/vector-im/compound/issues/194